### DISCUSSION TITLE
2.0 - Multi stream - multi cam

### DISCUSSION THREAD

**Discussion by CedricEugeni** [HaishinKit/HaishinKit.swift#1563] [Idea]
Hi,

In the new documentation for 2.0.0 beta-2 version, I saw that we can now have multiple streams at a time, which is great news. I also saw that we can set a track 0 or track 1 to a MTHKView to set which video track we want to display on which view, which is also great. 🎉 

What I'm looking for is: How can I send track 0 on RTMPStream 1 and track 1 on RTMPStream 2 ?

Thanks in advance for this discussion 🙏

**Comment by shogo4405** [comment]
I hadn't specifically considered that case. As of now, it can't be done. However, since it's not particularly difficult, I'll make sure it can be done.

https://github.com/shogo4405/HaishinKit.swift/blob/main/Sources/RTMP/RTMPStream.swift#L815-L820

**Reply by shogo4405** [reply]
I'll continue debugging until the official release, but it looks like it will turn out something like this. https://github.com/shogo4405/HaishinKit.swift/commit/0b7c99829aaa6098223a36be4641e22c7bd6450a

```swift
stream.selectTrack(0, mediaType: .video)
```

**Reply by CedricEugeni** [reply]
I think this seems perfect !
